setup.cfg

```diff
@@ -0,0 +1,7 @@
+[flake8]
+extend-ignore = E402
+exclude =
+    .git,
+    __pycache__
+max-complexity = 10
+max-line-length = 150
```

Comment:
Did we increase the max-line-length again? :upside_down_face: I thought we have a commong length of 120... in Python and in Rust... what do you think about it?